Status update 2022-06-10 (#99)
puppetlabs/content-and-tooling-team: content/blog/updates/2022-06-10-status-update.md

---
author: "LukasAud"
categories:
  - updates
date: "2022-06-10"
blog_tags: null
title: "2022-06-10: Content & Tooling Team Status Update"
---

## Yet another friday

Another week, another report. Except it's actually been two weeks!

Over the two community days we have released 11 modules, hitting a new record since the comeback of the CAT team and our weekly blogs.
Soon enough we are expecting all our modules to be up-to-date at the forge with all the upgrades they have been receiving.

We are also happy to announce that we've released a new version of PRM, 0.2.1. This new version includes some minor bugfixes and enhancements
that aim to improve the user experience. The full release notes can be found [here](https://github.com/puppetlabs/prm/releases/tag/0.2.1).

In addition to this, we have been working on a number of things, from implementing RedHat 9 into our supported modules to doing an overhaul of
the Apache module syntax, removing any legacy facts and ensuring that each variable has it's own type and description.

Finally, we have reworked some internal documentation to ensure that our team is ready to welcome new members in the future.

> That's all for this week, folks! We still have many projects in the drawing board so keep tuned in onto our weekly blogposts.

Reviewer: Change "so keep tuned in onto our" to "so keep tuned in to our"

---
author: "LukasAud"
categories:
  - updates
date: "2022-06-10"
blog_tags: null
title: "2022-06-10: Content & Tooling Team Status Update"
---

## Yet another friday

> Another week, another report. Except its actually been two weeks.  These weeks in Content and Tooling we have worked on:

Reviewer 2: ```suggestion
Another week, another report. Except it's actually been two weeks!
```

> On community days we have released 11 modules, hitting a new record since the comeback of the CAT team and our weekly blogs.

Reviewer 2: ```suggestion
Over the two community days we have released 11 modules, hitting a new record since the comeback of the CAT team and our weekly blogs.
```

Soon enough we are expecting all our modules to be up-to-date at the forge with all the upgrades they have been receiving.
But we still have a few bad boys out there that refuse to cooperate.

> On the development side of things, our tooling team member has been working on releasing a new version of PRM.

Reviewer 2: ```suggestion
We are also happy to announce that we've released a new version of PRM.
```

> This new 0.2.1 version includes some minor bugfixes and enhacements that aim to increase the QOL of its users.

Reviewer 2: ```suggestion
This new 0.2.1 version includes some minor bugfixes and enhancements that aims improve user experience.
```

> This new 0.2.1 version includes some minor bugfixes and enhacements that aim to increase the QOL of its users.

Reviewer: could add: "The full release notes can be found [here](https://github.com/puppetlabs/prm/releases/tag/0.2.1)"

In addition to this, we have been working on a number of things, from implementing RedHat 9 into our supported modules to

> completely rewriting the Apache module syntax, as well as working on several bugfixes for other modules, but those will have

Reviewer 3: maybe change this to `having done an overhaul of the Apache module syntax, removing any legacy facts and ensuring that each variable has it's own type and description,`

to wait until next week to see the daylight.

Finally, we have reworked some internal documentation to ensure that our team is ready to welcome new members in the future.

> That's all for this week, folks! It hasn't been that much of an exciting week but we still have many projects in the drawing board.

Reviewer 2: Don't think we need to highlight that it hasn't been exciting

- [`puppetlabs-docker`][puppetlabs-docker] (`4.4.0`)
- [`puppetlabs-apt`][puppetlabs-apt] (`8.4.0`)
- [`puppetlabs-java`][puppetlabs-java] (`8.1.0`)
- [`puppetlabs-java_ks`][puppetlabs-java_ks] (`4.3.1`)
- [`puppetlabs-reboot`][puppetlabs-reboot] (`4.2.0`)
- [`puppetlabs-motd`][puppetlabs-motd] (`6.2.0`)
- [`puppetlabs-registry`][puppetlabs-registry] (`4.1.0`)
- [`puppetlabs-sqlserver`][puppetlabs-sqlserver] (`3.1.0`)
- [`puppetlabs-package`][puppetlabs-package] (`2.2.0`)
- [`puppetlabs-exec`][puppetlabs-exec] (`2.1.0`)

  [puppetlabs-ntp]: https://github.com/puppetlabs/puppetlabs-ntp
  [puppetlabs-docker]: https://github.com/puppetlabs/puppetlabs-docker
  [puppetlabs-apt]: https://github.com/puppetlabs/puppetlabs-apt
  [puppetlabs-java]: https://github.com/puppetlabs/puppetlabs-java
  [puppetlabs-java_ks]: https://github.com/puppetlabs/puppetlabs-java_ks
  [puppetlabs-reboot]: https://github.com/puppetlabs/puppetlabs-reboot
  [puppetlabs-motd]: https://github.com/puppetlabs/puppetlabs-motd
  [puppetlabs-registry]: https://github.com/puppetlabs/puppetlabs-registry
  [puppetlabs-sqlserver]: https://github.com/puppetlabs/puppetlabs-sqlserver
  [puppetlabs-package]: 

>   [puppetlabs-exec]: 

Reviewer 2: Links need adding here

Reviewer 4: solved
